src/resolver.ts

```diff
@@ -40,7 +38,7 @@ export function handleAddrChanged(event: AddrChangedEvent): void {
   resolver.save()
 
   let domain = Domain.load(event.params.node.toHexString())
-  if(domain.resolver == resolver.id) {
+  if(domain && domain.resolver == resolver.id) {
```

Comment:
note: because setAddr can allow users to put any node, there is a situation that user specify non-existing domain